src/Microsoft.Framework.Runtime/project.json

```diff
@@ -2,18 +2,28 @@
     "version": "1.0.0-*",
     "description": "ASP.NET 5 runtime infrastructure for loading NuGet packages and projects.",
     "compilationOptions": { "define": [ "LOADER", "TRACE" ], "allowUnsafe": true, "warningsAsErrors": true },
+    "code": [
+        "**/*.cs",
+        "../../submodules/FileSystem/src/Microsoft.Framework.FileSystemGlobbing/**/*.cs"
```

Comment:
Preprocessor only flips public to internal, we still need to include the code if we want to use them.